types/network/Network.d.ts

```diff
@@ -344,14 +344,38 @@ export class Network {
   deleteSelected(): void;
 
   /**
-   * Returns the x y positions in canvas space of the nodes with the supplied nodeIds as an object.
+   * Returns the x y positions in canvas space of all nodes
    *
-   * Alternative inputs are a String containing a nodeId or nothing.
-   * When a String is supplied, the position of the node corresponding to the ID is returned.
-   * When nothing is supplied, the positions of all nodes are returned.
+   * @returns A an object containing the x y positions in canvas space of all nodes in the network, keyed by id.
    */
-  getPositions(nodeIds?: IdType[]): { [nodeId: string]: Position };
-  getPositions(nodeId: IdType): Position;
+  getPositions(): { [nodeId: string]: Position };
```

Comment:
I broke up getPositions() and getPositions(nodeIds: IdType[] | IdType) for my own sanity in documentation.

Reply:
Why? You could simply say that `nodeIds` defaults to ids of all the nodes currently in the network.

Also how can it return nothing? Doesn't it always return at least an empty object?

Reply:
Sure, I can do that.

You are correct on the nothing- I've revised the docs to indicate the an empty object will be returned. Next push should be correct in this regard.